application/akka/scala/AkkaQuickstart.scala

```diff
@@ -0,0 +1,23 @@
+//#full-example
+package example
+
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+
+//#main-class
+object AkkaQuickstart extends App {
+
+  if (sys.env.get("DEMO_USERNAME").getOrElse("").isEmpty() || sys.env.get("DEMO_PASSWORD").getOrElse("").isEmpty()) {
```

Comment:
Can you look into wrapping this in an HTTP server like the other examples?

Reply:
Done.